src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -1220,45 +1275,44 @@ void func_80A77844(EnIk* this, PlayState* play) {
     func_8002EBCC(&this->actor, play, 0);
     Gfx_SetupDL_25Opa(gfxCtx);
     Gfx_SetupDL_25Xlu(gfxCtx);
-    gSPSegment(POLY_OPA_DISP++, 0x08, func_80A761B0(gfxCtx, 245, 225, 155, 30, 30, 0));
-    gSPSegment(POLY_OPA_DISP++, 0x09, func_80A761B0(gfxCtx, 255, 40, 0, 40, 0, 0));
-    gSPSegment(POLY_OPA_DISP++, 0x0A, func_80A761B0(gfxCtx, 255, 255, 255, 20, 40, 30));
+    gSPSegment(POLY_OPA_DISP++, 0x08, EnIk_SetPrimColorEnvColor(gfxCtx, 245, 225, 155, 30, 30, 0));
+    gSPSegment(POLY_OPA_DISP++, 0x09, EnIk_SetPrimColorEnvColor(gfxCtx, 255, 40, 0, 40, 0, 0));
+    gSPSegment(POLY_OPA_DISP++, 0x0A, EnIk_SetPrimColorEnvColor(gfxCtx, 255, 255, 255, 20, 40, 30));
     SkelAnime_DrawFlexOpa(play, skelAnime->skeleton, skelAnime->jointTable, skelAnime->dListCount,
                           EnIk_OverrideLimbDraw2, EnIk_PostLimbDraw2, this);
```

Comment:
Could rename these LimbDraws based on their purpose now. For instance `EnIk_OverrideLimbDrawNabooruDefeat`, or I don't know if `OoT` uses the `System_SubSystem_Function` formating we use in MM, could do `EnIk_Nabooru_OverrideLimbDrawDefeat` where `Nabooru` is the subsystem.